src/containerapp/azext_containerapp/_params.py

```diff
@@ -209,6 +211,11 @@ def load_arguments(self, _):
         c.argument('name', id_part=None)
         c.argument('revision', help='Name of the revision.')
         c.argument('label', help='Name of the label.')
+        c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.')
```

Comment:
Please use the parameter name with clearer meanings like `--no-prompt`

Reply:
Could we define it as [`action='store_true'`](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags)?